test/CMakeLists.txt

```diff
@@ -19,6 +19,12 @@ endfunction()
 
 # Get rocRAND tests source files
 file(GLOB rocRAND_TEST_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+if(ROCRAND_CMAKE_HIP_LANGUAGE)
+    set_source_files_properties(${rocRAND_TEST_SRCS}
+      PROPERTIES
+        LANGUAGE HIP
+    )
+endif()
```

Comment:
It would be better to have these lines together with the lines for CUDA a few lines below.